src/main/java/virushade/Virushade.java

```diff
@@ -0,0 +1,39 @@
+package virushade;
+
+import virushade.tasks.TaskList;
+
+/**
+ * The Main class of the application.
+ */
+public class Virushade {
+    private Storage storageFile;
+    private TaskList taskList;
+
+    /**
+     * The constructor for Virushade.
+     * @param fileName The file path of the file where data is written to.
+     */
+    public Virushade(String fileName) {
+        storageFile = new Storage(fileName);
+        try {
+            TaskList taskList = new TaskList(storageFile);
+        } catch (VirushadeException e) {
+            Ui.handleVirushadeException(e);
+        }
+    }
+
+    /**
+     * The running sequence for Virushade.
+     */
+    public void run() {
+        Ui.interact();
+    }
```

Comment:
The Ui should not be responsible for all the backend work being done, it may be better to just focus the Ui on printing on the console and getting input from the console. Perhaps another class can be made to communicate between Ui, TaskList and Storage.